chain-impl-mockchain/src/transaction/witness.rs

```diff
@@ -217,7 +217,7 @@ impl Readable for Witness {
     }
 }
 
-#[cfg(test)]
+#[cfg(any(test, feature = "property-test-api"))]
 pub mod test {
```

Comment:
Why was this module public?
Can we have all unit test modules uniformly named `tests`?

Reply:
ok